Tests/NIOCoreTests/ByteBufferTest.swift

```diff
@@ -1754,14 +1754,24 @@ class ByteBufferTest: XCTestCase {
             throw XCTSkip("This test is only supported on 64-bit systems.")
```

Comment:
Can we now run this on 32-bit systems as well?

Reply:
Unfortunately no: we still need to be able to have a buffer size that is 32-bits long, but we store the sizes in `Int`.

Reply:
Unfortunately, the two `0xdeadbeef` lines you added below [broke compilation on 32-bit platforms like Android armv7 too](https://github.com/buttaface/swift-android-sdk/actions/runs/3909881597/jobs/6681462361#step:13:963), so this test will either need to be conditionally compiled only on 64-bit platforms or modified in some other way.

Reply:
Presumably the easiest fix is to just use a different sentinel value. The number doesn't matter: do you want to open a quick PR to use a different number?

Reply:
As in change it to a 32-bit number that could be stored in an Int32 also? It's your idea, please go ahead.

Reply:
Should be fixed by #2348.

Reply:
Yep, [compiles now](https://github.com/buttaface/swift-android-sdk/actions/runs/3917797023/jobs/6697815544), thanks.